### PR TITLE
Add default channel slug to env var

### PR DESCRIPTION
**Pull request by d-wysocki**
Related to multichannel changes:
- [saleor](https://github.com/mirumee/saleor/pull/6242)

Updated readme to drop old spectrum link

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-552 Share same DEFAULT_CHANNEL_ID in whole saleor-platform docker](https://app.clickup.com/t/2549495/SALEOR-552)

File: common.env [hunk 1]
```diff
@@ -2,3 +2,4 @@ DATABASE_URL=postgres://saleor:saleor@db/saleor
 DEFAULT_FROM_EMAIL=noreply@example.com
 CELERY_BROKER_URL=redis://redis:6379/1
 SECRET_KEY=changeme
+DEFAULT_CHANNEL_SLUG=default-channel
```
Comment (fowczarek) [inline common.env:5]: Lack of newline at end of the file.

**Comment by fowczarek** [comment]
Are you check that variable is visible in the storefront and dashboard?

**Comment by d-wysocki** [comment]
> Are you check that variable is visible in the storefront and dashboard?

We need to create frontend tasks for this. @fowczarek

**Review by fowczarek — APPROVED** [review]

**Pull request merged**